### DISCUSSION TITLE
<title> Artifacts won't leave Github Page after I git cloned the project in VSCode, even though I erased the files

### DISCUSSION THREAD

**Discussion by katelyngelle** [community/community#64570] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I clear my artifacts off of my site?

For some reason, I don't think VSCode is on the same account as my PC and laptop, so I just git cloned my page from my repository to my computer. I then deleted the unnecessary files (it's a blog page; I have two posts, post A and post B). However, the webpage now has 3 copies of Post A and 2 copies of Post B, and I don't know why.

I have cleared my browser cache, forced push by editing READ.ME, double-checked my commits, and waited for a couple hours, but the issue still persists. How do I remove these unnecessary post duplications? Thank you.

Also, I apologize if my explanation doesn't make any sense. I'm really new to coding.

**Comment by arafajs** [comment]
You clone all branches. Try this:
`git clone --single-branch --branch <branchname> <remote-repo>`
or:
`git clone -b <branch> <remote_repo>`

**Comment by LinukaAr** [comment]
It sounds like you've made changes to your blog repository and tried to update your webpage, but you're still seeing duplicate posts. Let's go through some steps to troubleshoot and resolve this issue:

1. **Check Repository:** Make sure you've actually deleted the unnecessary files from your repository. Sometimes, if you just delete the files locally on your computer, those changes won't reflect on GitHub until you commit and push those changes to your repository.

2. **Confirm Commits:** Double-check that you've committed and pushed your changes to the repository correctly. After making changes, you need to commit those changes locally and then push them to GitHub. Use the following commands in your terminal:

   ```
   git add .
   git commit -m "Deleted unnecessary files"
   git push origin master
   ```

3. **Check GitHub Pages Settings:** If your website is deployed using GitHub Pages, there might be some caching issues on GitHub's end. Go to your repository settings, specifically the GitHub Pages section, and check if there are any cached artifacts causing the duplication issue. You might find a "Clear cache" button or similar option.

4. **Wait for Propagation:** Sometimes, changes to websites hosted on GitHub Pages might take some time to propagate across all servers. This could be why you're still seeing duplicate posts even after waiting for a few hours.

5. **Check Jekyll Configuration:** If your blog is built using Jekyll (a static site generator), it might be a caching or build issue within the Jekyll configuration itself. Make sure you've properly configured the posts and categories in your Jekyll `_config.yml` or other relevant configuration files.

6. **Inspect Your Code:** Examine the source code of your generated webpage. Inspect the HTML and look for any anomalies that might be causing the duplicates. Ensure that your template code is correctly rendering the posts and not accidentally duplicating them.

7. **Force Refresh Your Browser:** While you mentioned clearing your browser cache, sometimes you might need to do a "hard refresh." On most browsers, you can do this by pressing `Ctrl + F5` (Windows/Linux) or `Cmd + Shift + R` (Mac).

8. **Rebuild Site Locally:** If you're using Jekyll or a similar static site generator, try rebuilding your site locally and checking if the duplicate posts still appear. This can help you determine if the issue is with your local setup or the GitHub Pages deployment.

9. **Reach Out for Help:** If you've tried all these steps and the issue still persists, consider reaching out to the GitHub Support community or the support channels of the specific static site generator you're using. They might be able to provide more targeted assistance based on the details of your setup.

Remember, troubleshooting issues like this can sometimes take time, especially if you're new to coding. Take it step by step, and don't hesitate to seek help from online communities and forums dedicated to your specific tools and technologies.
